src/main/java/com/graviteesource/policy/dynamicrouting/deployer/DynamicRoutingPolicyDeploymentLifecycle.java

```diff
@@ -0,0 +1,17 @@
+package com.graviteesource.policy.dynamicrouting.deployer;
+
+import io.gravitee.node.api.deployer.AbstractPluginDeploymentLifecycle;
+
+/**
+ * @author Kamiel Ahmadpour (kamiel.ahmadpour at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class DynamicRoutingPolicyDeploymentLifecycle extends AbstractPluginDeploymentLifecycle {
+
+    private static final String DYNAMIC_ROUTING_POLICY = "apim-policy-dynamic-routing";
```

Comment:
🤔 Just wondering. To ensure there is a single source of truth between those feature names and the one coming with the license file, should we declare those string constants in a module in `gravitee-license` ? Thought @gaetanmaisse @phiz71 ?